vmm/src/sigwinch_listener.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Alyssa Ross <hi@alyssa.is>
+// SPDX-License-Identifier: Apache-2.0
+
+use std::cell::RefCell;
+use std::fs::File;
+use std::io::{self, ErrorKind, Read, Write};
+use std::mem::size_of;
+use std::mem::MaybeUninit;
+use std::os::unix::prelude::*;
+use std::process::exit;
+use std::ptr::null_mut;
+
```

Comment:
nit: We don't normally split the `use` statements by source and instead let rustfmt just put them in alphabetical order. So if you remove these newlines then it will match the rest of the codebase.